src/extras/module-types.js

```diff
@@ -33,6 +35,9 @@
       if (cssContentType.test(contentType))
         return res.text()
         .then(function (source) {
+          source = source.replace(/url\(\s*(["'])?((?:\\.|[^\s,"'()\\])+)\1\s*\)/g, function (match, quotes, relUrl) {
+            return 'url(' + quotes + resolveUrl(relUrl, url) + quotes + ')';
+          });
```

Comment:
I just did some testing and spaces and brackets are of course permitted within quotes. For this reason I'd suggest the following adjustment:

```suggestion
          source = source.replace(/url\(\s*((["'])((?:\\.|[^\n\\"'])+)\2|((?:\\.|[^\s,"'()\\])+))\s*\)/g, function (match, quotes, relUrl1, relUrl2) {
            return 'url(' + quotes + resolveUrl(relUrl1 || relUrl2, url) + quotes + ')';
          });
```

Reply:
I tested it with font-awesome, but it didn't work.
```js
/url\(\s*((["'])((?:\\.|[^\n\\"'])+)\2|((?:\\.|[^\s,"'()\\])+))\s*\)/g
```
><img width="938" alt="Screen Shot 2021-05-02 at 11 25 59" src="https://user-images.githubusercontent.com/2867188/116801129-6fac9900-ab39-11eb-8e4e-14740b0a228b.png">
><img width="947" alt="Screen Shot 2021-05-02 at 11 29 14" src="https://user-images.githubusercontent.com/2867188/116801163-b7332500-ab39-11eb-96bd-c597cfff14ea.png">
><img width="942" alt="Screen Shot 2021-05-02 at 11 31 16" src="https://user-images.githubusercontent.com/2867188/116801198-fe211a80-ab39-11eb-8acc-d2f53d3b5de3.png">

So I made a little tweak, then it works.
```js
/url\(\s*(?:(["'])((?:\\.|[^\n\\"'])+)\1|((?:\\.|[^\s,"'()\\])+))\s*\)/g
```
><img width="948" alt="Screen Shot 2021-05-02 at 11 35 15" src="https://user-images.githubusercontent.com/2867188/116801248-94edd700-ab3a-11eb-82b3-268fecf6babf.png">
><img width="1071" alt="Screen Shot 2021-05-02 at 11 36 25" src="https://user-images.githubusercontent.com/2867188/116801260-b058e200-ab3a-11eb-97fa-694583f1a2a2.png">
><img width="1064" alt="Screen Shot 2021-05-02 at 11 37 14" src="https://user-images.githubusercontent.com/2867188/116801269-c9619300-ab3a-11eb-90aa-15f872a0c2d2.png">

Below are all the edge cases I have tested.
[https://regex101.com/r/uQkG3W/6](https://regex101.com/r/uQkG3W/6)